libfreerdp/core/activation.c

```diff
@@ -341,22 +342,25 @@ BOOL rdp_server_accept_client_font_list_pdu(rdpRdp* rdp, wStream* s)
 	    peer->AdjustMonitorsLayout(peer))
 	{
 		/* client supports the monitorLayout PDU, let's send him the monitors if any */
-		wStream* st = rdp_data_pdu_init(rdp);
-		BOOL r;
+		MONITOR_DEF* monitors = (MONITOR_DEF*) calloc(settings->MonitorCount, sizeof(MONITOR_DEF));
 
```

Comment:
Just curious, can't we have `settings->MonitorCount == 0`, because in this case we will end up with  ̀calloc`  returning NULL (and so error path) ?

Reply:
don't understand your question, the if
```c
	if (settings->SupportMonitorLayoutPdu && settings->MonitorCount && peer->AdjustMonitorsLayout &&
	    peer->AdjustMonitorsLayout(peer))

```

won't allow monitor count to be 0
and in this case the monitor layout pdu won't be sent
isn't this ok?

rebased on master

Reply:
ok that was my question. Merging.